docs/reference/glossary.md

```diff
@@ -0,0 +1,161 @@
+
+As a term, Proof-of-Storage refers to the design elements of the Filecoin protocol that allow one to guarantee (to some very high tolerance) that participants that claim to be contributing a given amount of storage are indeed fulfilling that pledge. In fact, Filecoin's Proof-of-Storage construction provides for a much stronger claim, allowing one to efficiently verify that a participant is storing a _particular piece of data_, without requiring that one have a copy of the file itself.
+
+*Note*: "proof" here is used in an informal sense - typically, these proofs take the form of a probabilistic argument, rather than a concrete proof; that is, it might _technically_ be possible to convince other participants that one is making a contribution one is not, but the possibility is so vanishingly slight as to border on impossibility.
+
+## Proof-of-Replication (PoRep)
+
+_Proof-of-Replication_ is a procedure by which a [storage miner](#storage-miner) can prove to the Filecoin network that they have created a unique copy of some piece of data on the network's behalf.
+
+## Proof-of-Spacetime (PoSt)
+
+_Proof-of-Spacetime_ is a procedure by which a [storage-miner](#storage-miner) can prove to the Filecoin network they continue to store a unique copy of some data on behalf of the network. Proof-of-Spacetime manifests in two distinct varieties in the present Filecoin specification: [Window Proof-of-Spacetime](#window-proof-of-spacetime-windowpost) and [Winning Proof-of-Spacetime](#winning-proof-of-spacetime-winningpost).
+
+## Quality-adjusted storage power
+
+The storage power a [storage miner](#storage-miner) earns from a storage deal offered by a [verified client](#verified-client) will be augmented by a multiplier. Power totals that take into account this multiplier are termed _quality adjusted_.
+
+## Retrieval miner
+
+A _retrieval miner_ is a Filecoin participant that enters retrieval [deals](#deal) with clients, agreeing to supply a client with a particular file in exchange for filecoin.  Note that unlike [storage miners](#storage-miner), retrieval miners are not additionally rewarded with the ability to add blocks to the Filecoin blockchain; their only reward is the fee they extract from the client.
+
+## Seal
+
+_Sealing_ is one of the fundamental building blocks of the Filecoin protocol. It is a computation-intensive process performed over a [sector](#sector) that results in a unique representation of the sector. The properties of this new representation are essential to the [Proof-of-Replication](#proof-of-replication-porep) and the [Proof-of-Spacetime](#proof-of-spacetime-post) procedures.
+
+## Sector
+
+Storage miners store data on behalf of the Filecoin network in fixed-size blocks of data called _sectors_. Currently, only 32GiB and 64GiB sectors are supported.
```

Comment:
> Currently, only 32GiB and 64GiB sectors are supported.

Smaller sectors (512MiB) are also supported in various devnets for faster testing & development. Here, I would either clarify that this is for Testnet only, or avoid size mentions here altogether.

Reply:
I took your advice and removed the mention altogether- after thinking about it, size mentions don't really help one understand what a sector is, and that information is easily discoverable when it is actually needed.